cpp/include/graph.hpp

```diff
@@ -60,19 +60,32 @@ class GraphViewBase {
   VT number_of_vertices;
   ET number_of_edges;
 
+  VT number_of_local_vertices;
+  ET number_of_local_edges;
+  VT local_offset;
+
   /**
    * @brief      Fill the identifiers array with the vertex identifiers.
    *
    * @param[out]    identifier      Pointer to device memory to store the vertex
    * identifiers
    */
   void get_vertex_identifiers(VT *identifiers) const;
+  void set_local_data(VT number_of_local_vertices_, ET number_of_local_edges_, VT local_offset_)
```

Comment:
can we pass and store arrays with this info for all ranks?

`void set_local_data(VT* number_of_local_vertices_, ET* number_of_local_edges_, VT* local_offset_)
`
so that we can access `number_of_local_vertices_[rank_i] ` from everywhere.

Reply:
We need this for communications like allgatherv

Reply:
Yes. Working on that

Reply:
Updated